packages/perseus-editor/src/components/link.jsx

```diff
@@ -8,21 +8,21 @@
 import {StyleSheet, css} from "aphrodite";
 import * as React from "react";
 
+import type {CSSProperties} from "aphrodite";
+
 // TODO(diedra): This is terrible! If we don't have a href for a link, it should
 // be a button instead. We just need to make sure we override the default browser
 // button styling (backgroundColor, border, and padding).
 const DEFAULT_HREF = "javascript:void(0)";
 
-type AphroditeStyle = any;
-
 type DefaultProps = {|
     element: string | React.AbstractComponent<any>,
     // Whether the link should appear in a 'highlighted' state. In
     // practice, this will apply the same styles that are applied on-hover.
     highlighted: boolean,
     href: string,
     // Pass through styles but also add our own to normalize
-    style: AphroditeStyle | Array<AphroditeStyle>,
+    style: CSSProperties | Array<CSSProperties>,
```

Comment:
Nit: probably want `$ReadOnlyArray<>` although I realize that's unrelated to what change you were making

Reply:
I think I missed porting that rule from webapp.  Could you take that on in a separate PR?